src/resmom_win/start_exec.c

```diff
@@ -4600,6 +4452,7 @@ set_credential(job *pjob, char **shell, char ***argarray)
 				name = lastname(*shell);
 				argv[i] = malloc(strlen(name) + 2);
 				if (argv[i] == NULL) {
+					log_err(-1, __func__, "argv is NULL");
```

Comment:
Please use errno instead of -1. Reword to something like "failed to malloc for argv"